.copier-answers.yml

```
@@ -1,5 +1,5 @@
 # Changes here will be overwritten by Copier
-_commit: 2.6.0
+_commit: 3.0.0
 _src_path: gh:DiamondLightSource/python-copier-template
```

```suggestion
_src_path: https://github.com/DiamondLightSource/python-copier-template
```

nit: If you want to automate these updates in the future, for now it's necessary to use the long form URL